wcfsetup/install/files/js/WoltLabSuite/Core/Controller/Clipboard.js

```diff
@@ -410,6 +415,11 @@ define(
 				
 				this._loadMarkedItems();
 			}).bind(this));
+
+			//noinspection JSUnresolvedVariable
```

Comment:
Bad location, this code is executed before the response from the server was received (see the code a few lines earlier).

Reply:
Which position would you recommend?